R/check_outliers.R

```diff
@@ -621,13 +618,10 @@ print.check_outliers <- function(x, ...) {
     }
 
     if ((isTRUE(nrow(outlier.count$all) > 0) || isTRUE(attributes(x)$grouped)) &&
-      (length(method) > 1 || all(method %in% method.univariate))) {
-      cat(long_dash,
-        "The following observations were considered outliers ",
-        "for two or more variables \n",
-        "by at least one of the selected methods: \n\n",
-        sep = ""
-      )
+          (length(method) > 1 || all(method %in% method.univariate))) {
+      cat(long_dash, "The following observations were considered outliers ",
+          "for two or more variables \n", "by at least one of the selected methods: \n\n",
```

Comment:
We could use `insight::format_alert()` here, which takes care of line breaks taking the terminal width into account.

Reply:
Ok I just tried it. Right now, the colour of the text (white, because I'm using a dark RStudio theme) matches the colour of the data frame. But when using `format_alert`, it changes the colour to blue. I would like it to be white though (or whatever default console colour is), since it's easier to read with lots of text as in our case (blue makes it look like a warning but it's literally the print output so that's weird). How can I change the colour using `format_alert`? I've tried `format_alert("This is a message.", color = "white")` but it's not working.

Reply:
Strange, this functions just calls `message()`, so no specific colour should be used except what your ide uses as default colour.

Reply:
My messages print in blue, that's why I used `cat` originally.

Reply:
@strengejacke OK, `format_message()` was what we were looking for, it does print correctly ;D